### ISSUE TITLE
What's the usage of add_time_ids?

### ISSUE THREAD

**Issue by WanrenZeng** [JyChen9811/FaithDiff#15]
It's a great work! I'm trying to finetune your model, but cannot understand the usage of your add_time_ids in the training script of the second stage.
In Line 920:
                # add cond
                add_time_ids = [
                    batch["original_size"].to(accelerator.device),
                    batch["crop_coords_top_left"].to(accelerator.device),
                    batch["target_size"].to(accelerator.device),
                ]
                add_time_ids = torch.cat(add_time_ids, dim=1).to(accelerator.device, dtype=weight_dtype)
                unet_added_cond_kwargs = {"text_embeds": pooled_text_embeds, "time_ids": add_time_ids}
            
                lq_img = batch["lq_images"].to(accelerator.device, dtype=weight_dtype)
                fake_hq_before_quant = unet.denoise_encoder(lq_img)
                noise_pred = unet(noisy_latents, timesteps, text_embeds, added_cond_kwargs=unet_added_cond_kwargs,  input_embedding=fake_hq_before_quant).sample

It seems to be a condition, but I didn't find the details about it in the paper.
I'd appreciate it if you could give me a explaination about what usage the "original_size", "crop_coords_top_left" and "target_size" will be. Thanks!

**Comment by JyChen9811** [comment]
Hi, the parameter settings are based on the SDXL [paper](https://arxiv.org/pdf/2307.01952) and the example script from [Diffusers](https://github.com/huggingface/diffusers/blob/5873377a660dac60a6bd86ef9b4fdfc385305977/examples/text_to_image/train_text_to_image_sdxl.py#L869C9-L869C25)).
